store/store.go

```diff
@@ -554,6 +554,18 @@ type requestOptions struct {
 
 // doRequest does an authenticated request to the store handling a potential macaroon refresh required if needed
 func (s *Store) doRequest(client *http.Client, reqOptions *requestOptions, user *auth.UserState) (*http.Response, error) {
+	// FIXME: find a better place for this?
```

Comment:
in its own function, and set on creation of `http.Client` in `newHTTPClient` and `download`?